web/src/contexts/SettingsContext.tsx

```diff
@@ -9,26 +9,26 @@ import {
 interface Settings {
   agentEndpoint: string
   runnerEndpoint: string
+  requireAuth: boolean
 }
 
 interface SettingsContextType {
-  getDefaultSettings: () => Settings
   settings: Settings
   updateSettings: (newSettings: Partial<Settings>) => void
+  getDefaultSettings: () => Settings
 }
 
-const getDefaultSettings = (): Settings => {
-  return {
-    agentEndpoint:
-      window.location.hostname === 'localhost'
-        ? 'http://localhost:8080'
-        : window.location.origin,
-    runnerEndpoint:
-      window.location.hostname === 'localhost'
-        ? 'ws://localhost:8080/ws'
-        : `${window.location.protocol === 'https:' ? 'wss:' : 'ws:'}//${window.location.host}/ws`,
-  }
-}
+const getDefaultSettings = (): Settings => ({
+  agentEndpoint:
+    window.location.protocol === 'https:'
```

Comment:
How come you changed the defaults to "example.com"? The previous code created sensible defaults. 
Was this intentional or test code that didn't get reverted?

Reply:
This tripped me up a lot; I kept misconfiguring it.

Reply:
Whooops. Sorry. That's what I get for using AI to rewrite some of this code. :-/